### PR TITLE
[12.0][MIG] purchase_order_line_sequence

### PR DESCRIPTION
**Pull request by bjeficent**
Migration of module purchase_order_line_sequence to version 12

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Isn't this a native feature now?

**Comment by AaronHForgeFlow** [comment]
@pedrobaeza I think you are right. However, this module does more things so I think it is also suitable to be ported:
- Propagates the same sequence to the stock move lines
- Shows the number in the PO line
- Shows the number in the report

**Comment by pedrobaeza** [comment]
The sequence is also propagated (not explicitly, but through record generation), and the other 2 features are noise IMO, but then please change readme properly.

**Comment by AaronHForgeFlow** [comment]
Yes better to update the readme. But the sequence is not propagated in standard, or at least, the sorting of the move lines is not the same as the sorting of the purchase order lines. I just tested on runbot: http://404354-12-0-96506d.runbot15.odoo.com/web?#id=37&action=747&model=purchase.order&view_type=form&menu_id=494

File: purchase_order_line_sequence/models/purchase.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                                   "displaying the purchase order.",
+                              default=9999)
+
+    sequence2 = fields.Integer('Sequence',
+                               help="Displays the sequence of the line in "
+                                    "the purchase order.",
+                               related='sequence', readonly=True)
+
+    @api.multi
+    def _create_stock_moves(self, picking):
```
Comment (pedrobaeza) [inline purchase_order_line_sequence/models/purchase.py]: Why not using `_prepare_stock_moves` method?

Comment (bjeficent) [reply purchase_order_line_sequence/models/purchase.py]: i didn't contemplate that solution. But you are right, this way is better

**Comment by HviorForgeFlow** [comment]
@mreficent travis is :red_circle:

**Review by AaronHForgeFlow — APPROVED** [review]
Works like a charm :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:

**Comment by AaronHForgeFlow** [comment]
@hveficent mr trvis is green now. Thanks @mreficent

**Review by AdriaGForgeFlow — APPROVED** [review]
Could you use this PR to update also Copyright from Eficent to Forgeflow?

Tested on runbot and seems to work as expected :+1:

**Comment by AaronHForgeFlow** [comment]
@pedrobaeza even if you will not use the module is it ok to you to merge it?

**Comment by tonygalmiche** [comment]
Beginner's question:
Why this module has the label 'merged' but it does not appear here:
https://github.com/OCA/purchase-workflow/issues/591
Thank you in advance

**Comment by HviorForgeFlow** [comment]
It is not automated, added btw

**Pull request merged**